R/calculate_boot_stats.R

```diff
@@ -29,49 +29,49 @@ calculate_boot_stats <- function(sample, rep, n = "auto", level = 0.95,
   if(!is.numeric(level)) {
     stop("level should be a numeric value")
   }
-  
+
   if(!length(level) == 1) {
     stop("level should only be length 1")
   }
-  
+
   if(!(level > 0 && level < 1 )) {
     stop("level should be between 0 and 1")
   }
-  
+
   if(!is.logical(pass_dist)) {
     stop("pass_dist should be logical (TRUE or FALSE)")
   }
-  
+
   if(!is.character(estimator)) {
     stop("Estimator should be a string")
   }
-  
+
   if(!(estimator %in% c("mean", "median", "var", "sd"))) {
     stop("Supported estimators are mean, median, var, sd")
   }
-  
+
   if(level < 0.7) {
     warning("Warning: chosen level is quite low--level is a confidence level,
             not a signficance level")
   }
-  
+
   # access the bootstrapping function
-  source("bootstrap.R")
-  
+  # source("bootstrap.R")
```

Comment:
Is this the only change to this file? 

I've addressed the source issue in my PR, I think we should not make changes in this PR to calculate_boot_stats here in order to avoid an unecessary merge conflict

Reply:
I uncommented it so it is back to being incorrect! Hopefully this means no merge conflicts for you!